### ISSUE TITLE
Vixen's gloves' trigger does not recognize Temporal Chains

### ISSUE THREAD

**Issue by Csiszi96** [PathOfBuildingCommunity/PathOfBuilding#6754]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### How is Path of Building expected to behave?

As I have in other PoB-s I expected the socketed curses in the Vixen's gloves to recognize another curse (Temporal Chains) and see that on the calculation page.

### How does Path of Building behave?

I pulled the settings of my character down via character import and PoB does not recognize Temporal Chains as a possible Vixen's trigger anymore. It does work if I swap it to some other curse spell though, like conductivity.

### How to reproduce the issue

I do not know, because if I make a new build by hand, it does behave as expected. 

### Character build code

```shell
https://pobb.in/enxA3Fl9o7mi
```

### Screenshots

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/26280306/a1f11ce9-e355-4495-9f34-c34fd9cba4d3)
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/26280306/9d7e7a40-0b69-4ee3-9bbe-bed90e8e7671)

**Comment by caffeinepills** [comment]
You didn't uncheck Punishment in your Vixen's, it's counting towards your 3 curses before your Temporal Chains.

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/17804711/ec3332a7-c641-43df-b564-4262c338e015)

You can see what 3 curses are being applied instead:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/17804711/67676747-dd30-4d24-b40e-0e778b62ad38)

**Comment by Csiszi96** [comment]
Also as I wrote in my ticket it does work with, for example, conductivity for me, so I would assume it is not the problem of the number of curses.

**Comment by Csiszi96** [comment]
That said, if I import my example PoB right now, it does seem to work, also I was looking around in another impending doom build I saved, and it had the same problem I described before, but if I export it into a URL and import it again, it does seem to work again. So actually I do not know how to reproduce the problem...

**Comment by Csiszi96** [comment]
I also noticed that if I save and reload a PoB it does not fix the problem, but if I close the application completely and load it in from a save it works again.

**Comment by caffeinepills** [comment]
No, that's not how it works. You have a 3 curse limit, which means 3 curses will be applied. You apply 4 curses and 1 gets overwritten, the first one in your glove, which is punishment. You aren't getting benefits from Punishment.

Loading your build and unchecking Punishment, it works just fine. Saving and re-opening, it's the same. The build you imported most likely had the same issue with Punishment since that's what all builds are doing. If unchecking Punishment didn't fix it, your install is bugged and I'd suggest uninstalling and reinstalling.

**Comment by Csiszi96** [comment]
My problem is not with punishment not applying, as I know it will get overwritten.  My problem is, as I stated previously, that Vixen's gloves does not recognize any triggers even though "Temporal Chains" is clearly available. And to reiterate one more time, it does work normally <ins>without</ins> unchecking punishment, and if I change "Temporal Chains" to another trigger like "Conductivity" in my previous example, it does work, meaning it has no relation to the curses socketed in the gloves, but had some kind of problem with "Temporal Chains". Also, as I commented before, reloading PoB seems to sometimes solve the issue. 
Moreover, when I had this issue (which I do not how to reproduce) unchecking punishment did not do anything for me.
